e2e-tests/e2eTest.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { describe, it, before } from 'node:test'
+import assert from 'node:assert/strict'
+import type { StackOutputs } from '../cdk/BackendStack.js'
+import { stackOutput } from '@nordicsemiconductor/cloudformation-helpers'
+import { STACK_NAME } from '../cdk/stackConfig.js'
+import { CloudFormationClient } from '@aws-sdk/client-cloudformation'
+import { DynamoDBClient } from '@aws-sdk/client-dynamodb'
+import { getRandomICCID } from './getRandomICCID.js'
+import { seedingDBFunction } from './seedingDBFunction.js'
+
+const CFclient = new CloudFormationClient()
+export const outputs = await stackOutput(CFclient)<StackOutputs>(STACK_NAME)
+export const db = new DynamoDBClient({})
+
+const iccidNew = getRandomICCID()
+const iccidOld = getRandomICCID()
+const iccidNotExisting = getRandomICCID()
+
+void describe('e2e-tests', () => {
+	before(async () => {
+		await seedingDBFunction(iccidNew, iccidOld, iccidNotExisting)
+	})
+	void it('should return statusCode 409 when the SIM information is not in DB', async () => {
+		const req = await fetchData(getRandomICCID())
+		assert.equal(req.status, 409)
+	})
+	void it('should return statusCode 400 when the SIM is not existing', async () => {
+		const req2 = await fetchData(iccidNotExisting)
+		assert.equal(req2.status, 400)
+	})
```

Comment:
`404` is used for not found: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404